bikespace_api/bikespace_api/api/submissions.py

```diff
@@ -38,7 +39,11 @@ def get_submission_with_id(submission_id):
 
 
 def get_submissions(request):
-    submissions = Submission.query.order_by(desc(Submission.parking_time)).all()
+    offset = request.args.get('offset', 1, type=int)
+    limit = request.args.get('limit', DEFAULT_OFFSET_LIMIT, type=int)
+
+    submissions = Submission.query.order_by(desc(Submission.parking_time)).paginate(offset, limit, False).items
```

Comment:
Could we return the `page`, `total` and `last` in the return json as well. it would be good to know how many items and pages are there in the query. 
https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/api/#flask_sqlalchemy.pagination.Pagination